src/cmd/configmap.rs

```diff
@@ -43,9 +43,7 @@ impl Executor for ConfigMap {
     #[cfg_attr(feature = "tracing", tracing::instrument(skip(config)))]
     async fn execute(&self, config: Arc<Configuration>) -> Result<(), Self::Error> {
         match self {
-            Self::Generate { namespace, name } => {
```

Comment:
This modification should not be part of this pull requests and are unnecessary, we use async here to keep consistency across the code base.